wp-directives.php

```diff
@@ -235,11 +238,24 @@ function wp_process_directives( $block_content ) {
 					continue;
 				}
 
-				call_user_func( $attribute_directives[ $attribute ], $tags, $context );
+				call_user_func(
+					$attribute_directives[ $attribute ],
+					$tags,
+					$context
+				);
 			}
 		}
 	}
 
 	return $block_content;
 }
-add_filter( 'render_block', 'wp_process_directives', 10, 1 );
+add_filter(
+	'render_block',
+	'wp_process_directives',
+	10,
+	1
+);
+
+// TODO: check if priority 9 is enough.
+// TODO: check if `wp_footer` is the most appropriate hook.
+add_action( 'wp_footer', WP_Directive_Store::render, 9 );
```

Comment:
Ah, don't we normally use a different notation for invoking a class method? 🤔 

```suggestion
add_action( 'wp_footer', array( 'WP_Directive_Store', 'render' ), 9 );
```

Reply:
I was sure there was a syntax for that, but I couldn't remember. 😄